Airports/EGTU/Regions

```diff
@@ -0,0 +1,279 @@
+; Start Dunkeswell (EGTU) Regions
+; - Runway
+REGIONNAME Dunkeswell
+smrSCORunway 0 N050.51.23.374 W003.14.24.657
```

Comment:
You can't have two definitions of a colour here, 'smrSCORunway' _and_ 0... either use (or add) a colour from [Colours.txt](https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Colours.txt), or just have the number. Not both :)